_docs/_latest/develop/dev/inventory.md

```diff
@@ -47,7 +47,7 @@ follow the process below:
     1. Specify if the pipeline will run.
 1. Flatten the data collected from your API so you can store it in a CSV or
     normalized storage system. For more information, see this example of
-    [flattening the data structure](https://github.com/GoogleCloudPlatform/forseti-security/blob/stable/google/cloud/security/inventory/pipelines/load_projects_pipeline.py#L32).
```

Comment:
@blueandgold This change makes the sentence nonsensical - it's saying "See this example" but then there isn't an example. Do we have one, or should we remove the sentence entirely?

Reply:
This sentence doesn't exist anymore with the rewrite.  So, nothing to do.